app-text/texlive/texlive-2021.recipe

```diff
@@ -6,27 +6,35 @@ are free software, including support for many languages around the world."
 HOMEPAGE="http://tug.org/texlive/"
 COPYRIGHT="1996 by collaboration between the TeX user groups."
 LICENSE="GNU GPL v2"
-REVISION="3"
-SOURCE_URI="ftp://tug.org/historic/systems/texlive/2014/texlive-20140525-source.tar.xz"
-SOURCE_URI_2="ftp://tug.org/historic/systems/texlive/2014/texlive-20140525-texmf.tar.xz"
-CHECKSUM_SHA256="08bb4eb02923ac65ce1183612c6b64da185904f26a1660f5ca4f902d055795be"
-CHECKSUM_SHA256_2="e0227251c5aa23f8b88973c298cc80b03bd238c645f7f82a6ce888a06b295963"
-SOURCE_DIR="texlive-20140525-source"
-SOURCE_DIR_2="texlive-20140525-texmf"
-PATCHES="texlive-2014.patchset"
-ADDITIONAL_FILES="tlutils.pm.patch"
-
-ARCHITECTURES="x86 !x86_gcc2 x86_64"
+REVISION="1"
+fullVersion=20210325
+SOURCE_URI="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-$fullVersion-source.tar.xz"
+SOURCE_URI_2="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-$fullVersion-texmf.tar.xz"
+SOURCE_URI_3="ftp://tug.org/historic/systems/texlive/$portVersion/install-tl-unx.tar.gz"
+SOURCE_URI_4="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-$fullVersion-tlpdb-full.tar.gz"
```

Comment:
Maybe rename the recipe's version to the fullVersion so you could use $portVersion instead?
`texlive-20210325.recipe` ,also done by some others as shown at repology: https://repology.org/project/texlive/versions

Reply:
I am not really convinced. The "official" version number is just the year, and we would need to hardcode it in the directory names of the URLs instead (where $portVersion is used now). Also, if you look at the SOURCE_DIR for install-tl, it uses a slightly different date.

Reply:
Didn't look that far, thanks for the explanation :)